### PR TITLE
Demo, Documentation, packages and code improvements

### PR DESCRIPTION
**Pull request by magemello**

### CODE REVIEW

File: ng2-components/ng2-alfresco-core/README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# Alfresco Angular 2 Components core
-
+# Alfresco Login Component for Angular 2
```
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-core/README.md]: Is that a copy-paste issue? Core is called Login.

File: ng2-components/ng2-alfresco-core/package.json [hunk 2]
```diff
@@ -63,13 +63,20 @@
     "@angular/router": "3.0.0",
     "@angular/upgrade": "2.0.0",
     "@types/node": "^6.0.42",
```
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-core/package.json]: Should be part of devDependencies

File: ng2-components/ng2-activiti-analytics/package.json [hunk 3]
```diff
@@ -41,51 +41,36 @@
...
     "rimraf": "2.5.2",
-    "traceur": "^0.0.91",
-    "tslint": "^3.8.1",
+    "traceur": "0.0.91",
+    "tslint": "3.15.1",
     "typescript": "^2.0.3",
-    "wsrv": "^0.1.5"
+    "wsrv": "^0.1.5",
+    "xo": "0.14.0",
+    "yargs": "4.7.0"
```
Comment (DenisVuyka) [inline ng2-components/ng2-activiti-analytics/package.json]: this is not needed, yargs is a node.js command line parser library (with 14 dependencies), we don't use it directly.

File: ng2-components/ng2-activiti-diagrams/package.json [hunk 4]
```diff
@@ -37,47 +37,31 @@
...
     "rimraf": "2.5.2",
-    "traceur": "^0.0.91",
-    "tslint": "^3.8.1",
+    "traceur": "0.0.91",
+    "tslint": "3.15.1",
     "typescript": "^2.0.3",
-    "wsrv": "^0.1.5"
+    "wsrv": "^0.1.5",
+    "xo": "0.14.0",
+    "yargs": "4.7.0"
```
Comment (DenisVuyka) [inline ng2-components/ng2-activiti-diagrams/package.json]: Not needed here, heavy library for command line arguments parsing (node.js).

**Comment by magemello** [comment]
ready

**Pull request merged**